src/towncrier/test/test_builder.py

```diff
@@ -52,7 +52,7 @@ def test_non_numeric_ticket_with_extension(self):
     def test_dots_in_ticket_name(self):
         self.assertEqual(
             parse_newfragment_basename("baz.1.2.feature", ["feature"]),
-            ("2", "feature", 0),
+            ("baz.1.2", "feature", 0),
```

Comment:
Again a bad test practice.
A test without a docstring.
But this is a nice test, as this specific feature is tested in isolation.

To me, this looks like a regression and this line should not be change.

But the docstring should be changed

```python
def test_dots_in_ticket_name(self):
    """
    The ticket ID can also be extracted from the last
    integer value found in the file name.
    """
```